From khali@linux-fr.org Sat Nov 26 23:58:45 2005
Date: Sun, 27 Nov 2005 08:57:10 +0100
From: Jean Delvare <khali@linux-fr.org>
To: "Mark M. Hoffman" <mhoffman@lightlink.com>, Greg KH <greg@kroah.com>
Cc: Mauro Carvalho Chehab <mchehab@brturbo.com.br>
Subject: [PATCH 09/25] i2c: Drop i2c_driver.flags, 3 of 3
Message-Id: <20051127085710.543e31ce.khali@linux-fr.org>
Content-Disposition: inline; filename=i2c-drop-driver-flags-03-flags.patch

The flags member of the i2c_driver structure is no more used. Drop it.

Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 include/linux/i2c.h |    3 ---
 1 file changed, 3 deletions(-)

--- gregkh-2.6.orig/include/linux/i2c.h
+++ gregkh-2.6/include/linux/i2c.h
@@ -112,7 +112,6 @@ struct i2c_driver {
 	char name[32];
 	int id;
 	unsigned int class;
-	unsigned int flags;		/* div., see below		*/
 
 	/* Notifies the driver that a new bus has appeared. This routine
 	 * can be used by the driver to test if the bus meets its conditions
@@ -250,8 +249,6 @@ static inline void i2c_set_adapdata (str
 	dev_set_drvdata (&dev->dev, data);
 }
 
-/*flags for the driver struct: */
-
 /*flags for the client struct: */
 #define I2C_CLIENT_ALLOW_USE		0x01	/* Client allows access */
 #define I2C_CLIENT_ALLOW_MULTIPLE_USE 	0x02  	/* Allow multiple access-locks */
